### PR TITLE
Ui pipeline state

### PR DESCRIPTION
**Pull request by lordmallam**
#### What does this PR do?
Create a pipeline with basic properties and route to mapping component
Setup redux actions and reducers for pipelines
Extract Pipeline config to separate route

#### How to test
Create pipelines
Notice they are stored in the redux store

#### Related JIRA ticket
[AET-153](https://jira.ehealthafrica.org/browse/AET-153)

### CODE REVIEW

File: aether-ui/aether/entrypoint.sh [hunk 1]
```diff
@@ -0,0 +1,202 @@
+#!/bin/bash
```
Comment (obdulia-losantos) [inline aether-ui/aether/entrypoint.sh]: Could you remove this file? The merge didn't take it out :see_no_evil:

**Comment by sabeen** [comment]
I am not in the position of reviewing the code, but creating a new piepline worked! 👍

File: aether-ui/aether/ui/assets/apps/components/views/pipelines/pipeline.jsx [hunk 2]
```diff
@@ -15,63 +16,97 @@ class PipeLine extends Component {
...
-      <div className={`pipeline pipeline--${this.state.pipelineView} ${this.state.showOutput ? 'show-output' : ''} ${this.state.fullscreen ? 'fullscreen' : ''}`}>
-        <div className='pipeline-nav'>
-          <div className='pipeline-nav-items'>
-            <div
-              className='pipeline-nav-item__input'
-              onClick={() => this.setState({ pipelineView: 'input' })}>
-              <div className='badge'>1</div>
-              Input
+      <div className={'pipelines-container show-pipeline'}>
+        <div className='navbar top-nav'>
```
Comment (obdulia-losantos) [inline aether-ui/aether/ui/assets/apps/components/views/pipelines/pipeline.jsx]: I think it's a good idea to take out the nav-bar of any component...

File: aether-ui/aether/ui/assets/apps/home.jsx [hunk 3]
```diff
@@ -14,9 +15,9 @@ const appElement = document.getElementById('home-app')
 const store = createStore()
 const component = (
   <Provider store={store}>
-    <div>
+    <IntlProvider locale='en'>
       <App />
-    </div>
+    </IntlProvider>
```
Comment (obdulia-losantos) [inline aether-ui/aether/ui/assets/apps/home.jsx:20]: :clap:

**Review by obdulia-losantos — APPROVED** [review]
We can begin the refactor task taking this as a starting point. :+1:

**Pull request merged**